### DISCUSSION TITLE
Disabling the dictionary view when editing a value that is a foreign key.

### DISCUSSION THREAD

**Discussion by jassuncao** [dbeaver/dbeaver#35571] [Q&A]
When using the grid view and a cell that is a foreign key is selected, the value panel will be filled with possible values. The values seem to be retrieved using a background task/query that can have a serious impact if the target the association has a large number of rows. 
Example of the query used:
`SELECT id dbvrvalue, id FROM tzc_corerepository."transaction" WHERE id < $1 ORDER BY id DESC`
You can imagine the impact of this query when the table in question has millions of rows...
Is it possible to disable this behavior ?

**Comment by E1izabeth** [comment]
Perhaps you can just close/not open Value panel

**Reply by jassuncao** [reply]
Yes, I'm doing that as an workaround. Making a connection read-only is another possibility.
But the Value panel is  very useful, for example to view and edit json columns, so I would prefer to keep using it.
One of this days a production server came to a crawl when I forgot to close the panel and accidentally clicked on column with an association to a table with a couple of millions of rows. :cold_sweat:

**Reply by E1izabeth** [reply]
Could you create an issue about that?

**Reply by jassuncao** [reply]
Will do that.
